symphony/app/fbcnms-projects/magmalte/app/components/TabbedTable.js

```diff
@@ -36,62 +89,93 @@ type TabPanelProps = {
-                        <TableCell key={rowIdx + '-' + cellIdx}>
-                          {cellItem}
-                        </TableCell>
-                      );
-                    })}
-                  </TableRow>
-                );
-              })}
-            </TableBody>
-          </Table>
-        </TableContainer>
-      ) : null}
-    </>
-  );
+  if (itemData.length > 0) {
+    return (
+      <>
+        {currTabIndex === index ? (
+          <TableContainer component={Paper} elevation={0}>
+            <Table>
+              <TableBody>
+                {itemData.map((rowItem, rowIdx) => {
+                  return (
+                    <TableRow key={rowIdx} data-testid={'alertName' + rowIdx}>
+                      <TableCell component="th" scope="row">
+                        <Text variant="body3" className={classes.rowTitle}>
+                          {rowItem.name}
+                        </Text>
+                      </TableCell>
+                      {rowItem.cols.map((cellItem, cellIdx) => {
+                        return (
+                          <TableCell key={rowIdx + '-' + cellIdx}>
+                            <Text variant="body3" className={classes.rowText}>
+                              {cellItem}
+                            </Text>
+                          </TableCell>
+                        );
+                      })}
+                    </TableRow>
+                  );
+                })}
+              </TableBody>
+            </Table>
+          </TableContainer>
+        ) : null}
+      </>
+    );
+  } else {
+    return currTabIndex === index ? (
+      <Paper elevation={0}>
+        <Grid
+          container
+          alignItems="center"
+          justify="center"
+          xs={12}
+          className={classes.emptyTable}>
+          <Grid item className={classes.emptyTableContent}>
+            <Text variant="body2">You have 0 {label} Alerts</Text>
+            <Text variant="body3">
+              To add alert triggers click <Link href="#">alert settings</Link>.
+            </Text>
+          </Grid>
+        </Grid>
+      </Paper>
+    ) : null;
+  }
 }
 
 export default function TabbedTable(props: Props) {
+  const classes = useStyles();
   const [currTabIndex, setCurrTabIndex] = useState<number>(0);
   const tabPanel = Object.keys(props.data).map((k: string, idx: number) => {
     return (
       <TabPanel
         key={idx}
         index={idx}
+        label={k}
         currTabIndex={currTabIndex}
         itemData={props.data[k]}
       />
     );
   });
   return (
     <>
-      <Tabs
+      <MagmaTabs
         value={currTabIndex}
         onChange={(_, newIndex: number) => setCurrTabIndex(newIndex)}
-        indicatorColor="primary"
-        textColor="primary"
         variant="fullWidth">
         {Object.keys(props.data).map((k: string, idx: number) => {
-          return <Tab key={idx} label={k} />;
+          return (
+            <MagmaTab
```

Comment:
same here, can spare the return statement :)

Reply:
yes, you have to remove the return and curly braces, these two blocks of code are equivalen:
```
const rows = data.map(i => (
  <div>{I}</div>
));

const rows = data.map(i => {
  return (
    <div>{I}</div>
  );
});
```